astroquery/utils/tap/model/job.py

```diff
@@ -124,12 +125,12 @@ def __change_phase(self, phase, verbose=False):
                 phase = 'QUEUED'
                 if response.status != 200 and response.status != 303:
                     errMsg = taputils.get_http_response_error(response)
-                    print(response.status, errMsg)
+                    log.error(response.status, errMsg)

```

Comment:
this seems to be repetitive here and below, but keep them in for now, removing the duplicates can be part of the cleanup of tap.